app/views/home.njk

```diff
@@ -44,11 +44,11 @@
             </thead>
             <tbody class="govuk-table__body">
 
-              {% for message in messageData %}
+              {% for notification in notificationData %}
 
               <tr class="govuk-table__row action"> 
-                <td class="govuk-table__cell"><a class="govuk-link--no-visited-state" href="/messages/message/{{ message.messageId }}">{{ message.content.heading }}</a></td>
-                <td class="govuk-table__cell">{{ message.requestedDate }}</td>
+                <td class="govuk-table__cell"><a class="govuk-link--no-visited-state" href="/messages/message/{{ notification.messageId }}">{{ notification.content.heading }}</a></td>
```

Comment:
Agree we need to consider if it makes sense to update the db to reflect the change to notications instead of messages.

alternatively we could just have an `id` column?

Reply:
Yeah `id` might just be the best way to go, guess we'll see how we go about naming columns on other tables? Will keep this in mind.